integreat_cms/cms/views/analytics/app_size_view.py

```diff
@@ -29,6 +28,7 @@ def get_context_data(self, **kwargs: Any) -> dict:
         """
         context = super().get_context_data(**kwargs)
 
+        # pylint: disable=fixme
```

Comment:
I don't think we need this comment anymore :D I hope/think that we have been using the correct calculation for a long time already :)